homeassistant/components/stream/manifest.json

```diff
@@ -4,6 +4,7 @@
   "codeowners": ["@hunterjm", "@uvjustin", "@allenporter"],
   "dependencies": ["http"],
   "documentation": "https://www.home-assistant.io/integrations/stream",
+  "import_executor": true,
```

Comment:
Is this still needed if the `import av` is already run inside the executor ?

Reply:
It can import numpy and PyTurboJPEG as well
  "requirements": ["PyTurboJPEG==1.7.1", "ha-av==10.1.1", "numpy==1.26.0"]